ledger/ledger-on-memory/src/main/scala/com/daml/ledger/on/memory/InMemoryState.scala

```diff
@@ -50,5 +50,5 @@ object InMemoryState {
   type MutableState = mutable.Map[StateKey, StateValue] with ImmutableState
 
   type StateKey = ByteString
-  type StateValue = Array[Byte]
+  type StateValue = ByteString
```

Comment:
I guess it makes sense to use `Bytes` here too?

Reply:
Yes, indeed -- changed.